ui/src/shared/apis/query.ts

```diff
@@ -21,7 +28,7 @@ export const fetchTimeSeries = async (
   queries: Query[],
   resolution: number,
   templates: Template[],
-  editQueryStatus: () => any
+  editQueryStatus: () => any = noop
```

Comment:
Why are we setting this equal to noop?

Reply:
When updating it to take on the responsibilities of `fetchTimeSeriesAsync` I borrowed the default to`noop` for `editQueryStatus` behavior. Although, it's a good question because I think `editQueryStatus` was only missing when called in the `Table`.  If that's the case then I can import `noop` there and remove this default.

Reply:
Actually, that change broke dashboards when I removed the `noop` default. I'll look into why. I think the issue is in `AutoRefresh`